src/Components/Auth/Login.tsx

```diff
@@ -190,7 +190,7 @@ export const Login = (props: any) => {
                 )}
               </div>
             </CardContent>
-            <CardActions className="padding16">
+            <CardActions style={{ padding: "16px" }}>
```

Comment:
As an alternative to inline css, Can we use tailwind classes??

Reply:
Nope, We can't use tailwind classes for MUI components. 
If we want to use tailwind classes, then we need to replace `mui` with `material-tailwind` package